src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -600,4 +612,16 @@ class JesBackend extends Backend with LazyLogging with ProductionJesAuthenticati
   override def findResumableExecutions(id: WorkflowId)(implicit ec: ExecutionContext): Future[Map[ExecutionDatabaseKey, JobKey]] = {
     globalDataAccess.findResumableJesExecutions(id)
   }
+
+  /**
+    * Will try to instantiate a user authenticated IOInterface if possible.
+    * If not, will default back to the Cromwell authenticated IOInterface.
+    */
```

Comment:
Clearly I'm not completely up to date on our current auth scheme, why is this desirable behavior?